core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java

```diff
@@ -205,4 +195,210 @@ public JsonObject exportSystemErrors() {
     }
     return noticesByType;
   }
+
+  /**
```

Comment:
If this is a public data container class, it seems like it would be better off in the `.model` package. This would greatly simplify NoticeContainer and it also makes it clearer that it's a general data container class that's used outside of NoticeContainer (e.g., in the output comparator module). When something is an inner class it generally means it's always used in context of the outer class.